addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1354,7 +1354,8 @@ export function isMediaElement(node) {
     );
 }
 export function isVoidElement(node) {
-    return isMediaElement(node) || node.tagName === 'HR';
+    return isMediaElement(node) || node.tagName === 'HR' || node.tagName === 'IMG';
+    // Not sure why it's useful, can't reproduce issue with IMG nodes
```

Comment:
@dmo-odoo I can confirm the flows we discussed in this PR, for both snippets, work without this code. Therefore it seems unrelated and I would need an explanation to write a decent commit message.
Maybe it should even be in a separate commit/PR ?

Reply:
Had to remove it because it made tests fail. I'll ask DBO on the task why this is in the spec.

Reply:
Removing it was the right call. We reached the same conclusion in another case with RCDL, see https://github.com/odoo/odoo/pull/113379#discussion_r1173370261

Reply:
It is once more needed (at least if we don't want to add many more conditions/exceptions). @qsm-odoo any idea about the failing tests ?